src/geode/io/model/msh_output.cpp

```diff
@@ -441,5 +443,26 @@ namespace geode
             MSHOutputImpl impl( filename(), brep );
             impl.write_file();
         }
+
+        bool MSHOutput::is_saveable( const BRep& brep ) const
+        {
+            for( const auto& surface : brep.surfaces() )
+            {
+                if( surface.mesh().type_name()
+                    != TriangulatedSurface3D::type_name_static() )
```

Comment:
Since MSH is able to store non simplicial elements, this limitation should be found somewhere in our doc.

Reply:
Yes, but where?